modelina-website/src/components/layouts/Head.tsx

```diff
@@ -18,9 +18,9 @@ export interface HeadProps {
 export default function HeadComponent({
   title,
   description = 'Open source tools to easily build and maintain your event-driven architecture. All powered by the AsyncAPI specification, the industry standard for defining asynchronous APIs.',
-  image = '/img/social/website-card.jpg'
+  image = '/img/card/website-card.jpg'
 }: HeadProps) {
-  const url = process.env.DEPLOY_PRIME_URL || process.env.DEPLOY_URL;
+  const url = 'https://modelina.org';
```

Comment:
We can set an environment variable for this instead of directly mentioning it here and use it as it was used earlier.

cc: @jonaslagoni

Reply:
@devilkiller-ag Make sure to start with NEXT_PUBLIC_ , if you want to include the environment variable on the bundle during the build time

https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser

Reply:
👍

Reply:
I have added the environment variable on the Netlify.